src/api/lib/tasks/project_log_rotate_manually.rake

```diff
@@ -1,20 +0,0 @@
-desc 'Run project log rotate job manually'
```

Comment:
The commit message states that "It doesn't make sense any more", but it doesn't say why. Please add a short explanation to the commit.